src/main/scala/org/broadinstitute/dsde/workbench/sam/api/UserRoutes.scala

```diff
@@ -45,6 +45,15 @@ trait UserRoutes extends UserInfoDirectives {
       requireUserInfo { userInfo =>
         asWorkbenchAdmin(userInfo) {
           pathPrefix("user") {
+            path("email" / Segment) { email =>
+              complete {
+                userService.getUserStatusFromEmail(email).map { statusOption =>
+                  statusOption.map { status =>
+                    StatusCodes.OK -> Option(status)
+                  }.getOrElse(StatusCodes.NotFound -> None)
```

Comment:
what does None actually look like serialized to json?

Reply:
it's an empty response (and great question, I didn't think about it, copied this from `/api/admin/user/{userId}`)